src/Xamarin.Android.Build.Tasks/Tasks/CreateMultiDexMainDexClassList.cs

```diff
@@ -51,7 +51,7 @@ public override bool Execute ()
 			EnvironmentVariables =
 				string.IsNullOrEmpty (MSBuildRuntimeType) || MSBuildRuntimeType == "Mono" ?
 				new string [] { "PROGUARD_HOME=" + ProguardHome } :
-				new string [] { "JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF8", "PROGUARD_HOME=" + ProguardHome };
+				new string [] { "JAVA_TOOL_OPTIONS=-Dfile.encoding=UTF8" };
```

Comment:
This will need to be reverted when PR #386 is merged. Please add a TODO comment noting this, so that we don't forget.